src/main/java/VASSAL/build/module/map/KeyBufferer.java

```diff
@@ -123,13 +123,13 @@ public void mousePressed(MouseEvent e) {
     if (p != null && !ignoreEvent) {
       boolean movingStacksPickupUnits = (Boolean) GameModule.getGameModule().getPrefs().getValue(Map.MOVING_STACKS_PICKUP_UNITS);
       if (!kbuf.contains(p)) {
-        if (!e.isShiftDown() && !e.isControlDown()) {
+        if (!e.isShiftDown() && !SwingUtils.isControlDown(e)) {
           kbuf.clear();
         }
         // RFE 1629255 - If the top piece of an unexpanded stack is left-clicked
         // while not selected, then select all of the pieces in the stack
         // RFE 1659481 - Control clicking only deselects
-        if (!e.isControlDown()) {
+        if (!SwingUtils.isControlDown(e)) {
```

Comment:
Must disable on Mac -- eventually through using Command button, but for now at least not allowing Control through